.all-contributorsrc

```diff
@@ -238,6 +238,17 @@
       "contributions": [
         "example"
       ]
+    },
+    {
+      "login": "Dajust",
+      "name": "Justice Mba ",
```

Comment:
There's an extra space after your name, typo?

Reply:
Not sure, I think that file is auto-generated.

Reply:
You can fix it manually.

You may want to fix it on GitHub too though :wink:

<img width="148" alt="screen shot 2018-06-12 at 11 59 51 am" src="https://user-images.githubusercontent.com/1500684/41308081-2a4b1aa8-6e38-11e8-88be-0a2651d78127.png">